pkg/oci/spec.go

```diff
@@ -115,22 +115,20 @@ func ApplyOpts(ctx context.Context, client Client, c *containers.Container, s *S
 	return nil
 }
 
+// defaultUnixCaps is the set of the default caps.
+// The following caps are disabled by default since containerd v2.0:
+// MKNOD, NET_RAW, SETFCAP, SYS_CHROOT, AUDIT_WRITE.
 func defaultUnixCaps() []string {
 	return []string{
 		"CAP_CHOWN",
 		"CAP_DAC_OVERRIDE",
 		"CAP_FSETID",
 		"CAP_FOWNER",
-		"CAP_MKNOD",
```

Comment:
q: If `MKNOD` is removed, will it result in containers that mount `/dev/sdX` without privileged mode to fail?

Reply:
Dealing with devs need securityContext.privileged anyway

Reply:
That is if you use via CRI right? I was thinking of cases where you run the container directly using tools like docker or nerdctl.

Reply:
Do you mean `docker run --device /dev/sdX`? Why does it need `CAP_MKNOD`?

Reply:
I was under the impression that to create the device within the container, the capability was needed.

Reply:
`docker run --device /dev/sda --cap-drop MKNOD alpine hexdump /dev/sda` seems to work

Reply:
Question: Then is the capability added/dropped after the devices are created?

Reply:
Yes, as the devices are created by runc, not by the containerized workload
https://github.com/opencontainers/runc/blob/v1.1.13/libcontainer/rootfs_linux.go#L99

Reply:
(In theory, there can be a program that mknods the device node in a non-standard path by itself, though. Such a program will need `--cap-add MKNOD` explicitly)

Reply:
Wouldn't this break fifo's?